scraper.py

```diff
@@ -0,0 +1,46 @@
+__author__ = 'shrenilpatel'
+
+import json
+import urllib2
+
+#FOOD SERVICE
+getUrl = "https://api.uwaterloo.ca/v2/foodservices/locations.json?key=a19ad7d4ee67feebd6b7caa461251302"
+#postUrl = 'https://geotaganything.herokuapp.com/tags' #ACTUAL
```

Comment:
I'm assuming posting to geotaganything.herokuapp.com won't work for this script due to authentication -- that's why it's commented out. Correct?